privacyidea/api/lib/prepolicy.py

```diff
@@ -694,8 +694,11 @@ def check_max_token_user(request=None, action=None):
     params = request.all_data
     serial = getParam(params, "serial")
     user_object = get_user_from_param(params)
-    if not user_object.login and serial:
-        user_object = get_token_owner(serial)
+    if user_object.is_empty() and serial:
+        try:
+            user_object = get_token_owner(serial) or User()
+        except:
+            user_object = User()
```

Comment:
I think user_object is already an empty User() if an exception occurs (see line 697). So imho there is no need to set the user_object ot an empty user again. You could simply do a ``pass``.

Is it possible to determine, which Exception we would catch, so that we only would do ``except ExeptionType``?

Reply:
That's correct. Thank you. The exeption Type is ResourceNotFoundError.